Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py

```diff
@@ -150,205 +106,219 @@ def get_jobs(client: Client) -> CommandResults:
-        title = event.get("name")
-        severity = event.get("severity")
-
-        body = client.map_to_arcanna_raw_event(job_id, event, severity, title)
-        body["label"] = client.map_to_arcanna_label(arcanna_label, closing_notes, closing_user)
-
-        mapped_events.append(body)
-
-    response = client.send_bulk(job_id, mapped_events)
-    readable_output = f' ## Arcanna send bulk results: {response}'
+def send_event_feedback(client: Client, default_job_id: int, args: Dict[str, Any]) -> CommandResults:
+    job_id = args.get("job_id", default_job_id)
+    event_id = args.get("event_id")
+    username = args.get("username")
+    feedback = args.get("feedback", "<empty>")
+    decision_set = args.get("valid_decisions", [])
+    
+    if feedback in decision_set is False:
+        raise Exception(f"Error in arcanna-send-event-feedback.Unknown decision - {arcanna_label} ; Valid options: {decision_set} ")
+
+    payload = map_to_arcanna_label(feedback, username)
+    
+    result = client.send_feedback(job_id, event_id, payload)
+    
+    headers = ["feedback_status", "status", "details"]
+    readable_output = tableToMarkdown(name="Arcanna Event", headers=headers, t=result)
+    outputs = {
+        'Arcanna.Event(val.job_id && val.job_id === obj.job_id)': createContext(result)
+    }
 
     return CommandResults(
         readable_output=readable_output,
-        outputs_prefix='Arcanna.Bulk',
-        outputs_key_field='status',
-        outputs=response
+        outputs=outputs,
+        raw_response=result
     )
 
 
-def parse_mappings(mapping: str) -> dict:
-    result = {}
-    pairs = mapping.split(",")
-    for pair in pairs:
-        parts = pair.split("=")
-        if len(parts) != 2:
-            raise BaseException("Arcanna: Error while parsing mapping fields")
-        demisto_closing_reason = parts[0].strip().replace("\"", "")
-        arcanna_label = parts[1].strip().replace("\"", "")
-        result[demisto_closing_reason] = arcanna_label
-
-    return result
-
+''' MAIN FUNCTION '''
 
 def main() -> None:
-    """main function, parses params and runs command functions
-
-    :return:
-    :rtype:
-    """
-
-    api_key = demisto.params().get('apikey')
-
-    # get the service API url
-    base_url = urljoin(demisto.params()['url'])
-    verify_certificate = not demisto.params().get('insecure', False)
-    feature_mapping = demisto.params().get('feature_mapping')
-    proxy = demisto.params().get('proxy', False)
+	"""main function, parses params and runs command functions
 
-    default_job_id = demisto.params().get('default_job_id', -1)
-    demisto.debug(f'Command being called is {demisto.command()}')
-    try:
+	:return:
+	:rtype:
+	"""
 
-        client = Client(
-            api_key=api_key,
-            base_url=base_url,
-            verify=verify_certificate,
-            proxy=proxy,
-            default_job_id=default_job_id
-        )
+	api_key = demisto.params().get('credentials', {}).get('password')
```

Comment:
You can do something like this to not break BC:
api_key = demisto.params().get('credentials', {}).get('password') or demisto.params().get('apikey')